test/functional/json/jsonb.ts

```diff
@@ -34,4 +34,27 @@ describe("jsonb type", () => {
         expect(foundRecord!.data.foo).to.eq("bar");
     })));
 
+    it("should persist jsonb string correctly", () => Promise.all(connections.map(async connection => {
+        await connection.syncSchema(true);
+        let recordRepo = connection.getRepository(Record);
+        let record = new Record();
+        record.data = JSON.stringify({ foo: "bar" });
```

Comment:
Why do you stringify json here? I think `record.data` should access an object since its "json" column.